modules/tinymce/src/plugins/table/test/ts/browser/ui/TableBorderStyleTest.ts

```diff
@@ -0,0 +1,196 @@
+import { Assertions, Keys, Waiter } from '@ephox/agar';
+import { afterEach, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections, TinyUiActions } from '@ephox/mcagar';
+import { SugarBody, SugarElement } from '@ephox/sugar';
+import Editor from 'tinymce/core/api/Editor';
+import { EditorEvent } from 'tinymce/core/api/util/EventDispatcher';
+import { TableModifiedEvent } from 'tinymce/plugins/table/api/Events';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.plugins.table.ui.TableBorderStyleTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'table',
+    indent: false,
+    toolbar: 'tablecellborderstyle',
+    base_url: '/project/tinymce/js/tinymce',
+    setup: (editor: Editor) => {
+      editor.on('tablemodified', logEvent);
+    },
+    table_border_styles: [
+      {
+        title: 'Solid',
+        value: 'solid'
+      },
+      {
+        title: 'None',
+        value: ''
+      },
+    ]
+  }, [ Plugin, Theme ], true);
+
+  let events: Array<EditorEvent<TableModifiedEvent>> = [];
+  const logEvent = (event: EditorEvent<TableModifiedEvent>) => {
+    events.push(event);
+  };
+
+  afterEach(() => {
+    events = [];
+  });
```

Comment:
This is never used in this test, so why are we logging the events? I'm not sure if this should be removed or if we should be asserting the events in the tests.

- [ ] Remove or add an assertion

Reply:
Because I copy-pasted it, not thinking.